pages/sponsors.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Navbar from './../Components/navbar.js'
+import Body from './../Components/body.js'
+import { createMuiTheme, makeStyles, ThemeProvider } from '@material-ui/core/styles';
+
+export default function Sponsors() {
+  return (
+    <body style={{margin: "0"}}>
+      <div>
+        <Navbar/>
+			<h1> Sponsors </h1>
```

Comment:
fix indent here

Reply:
Indentations are getting worse as I try to fix them. Look fine on my end but on github they aren't. If someone else could take a look and try fixing it that'd be appreciated.